### PR TITLE
Version 3

### PR DESCRIPTION
**Pull request by pollen8**
Merge in version 3 code to master
Add Chord and brush components:

image
image

### CODE REVIEW

File: src/legacy/TornadoD3.ts [hunk 1]
```diff
@@ -284,11 +280,11 @@ export class TornadoD3 {
...
       .call(yAxis);
-    attrs(svg?.selectAll('.y-axis .domain, .y-axis .tick line'), axis.y.style);
-    attrs(svg?.selectAll('.y-axis .tick text'), axis.y.text.style as any);
+    // attrs(svg?.selectAll('.y-axis .domain, .y-axis .tick line'), axis.y.style);
+    // attrs(svg?.selectAll('.y-axis .tick text'), axis.y.text.style as any);
 
-    attrs(svg?.selectAll('.x-axis .domain, .x-axis .tick line'), axis.x.style);
-    attrs(svg?.selectAll('.x-axis .tick text'), axis.x.text.style as any);
+    // attrs(svg?.selectAll('.x-axis .domain, .x-axis .tick line'), axis.x.style);
+    // attrs(svg?.selectAll('.x-axis .tick text'), axis.x.text.style as any);
```
Comment (annaet) [inline src/legacy/TornadoD3.ts:287]: Are these intentionally commented out?

File: src/ScatterPlot.tsx [hunk 2]
```diff
@@ -1,127 +1,134 @@
...
-    }
-  }
-
-  public render() {
-    return <div ref={(ref) => this.ref = ref} className="scatterplot-chart-container"></div>;
-  }
-}
-
-export default ScatterPlot;
+// import { extent } from 'd3-array';
```
Comment (annaet) [inline src/ScatterPlot.tsx:1]: Is this file intentionally commented out?

File: src/utils/useMakeLine.ts [hunk 3]
```diff
@@ -11,7 +11,8 @@ import {
   useState,
 } from 'react';
 
-import { IAxes } from '../Histogram';
+import { IAxes } from '../legacy/types';
+// import { IAxes } from '../components/YAxis';
```
Comment (annaet) [inline src/utils/useMakeLine.ts:15]: Should this be removed?

Comment (pollen8) [reply src/utils/useMakeLine.ts:15]: removed :)

File: src/legacy/TornadoD3.ts (hunk 1 above)
Comment (pollen8) [reply src/legacy/TornadoD3.ts]: nope :) I've put them back in now thanks for catching that

File: src/ScatterPlot.tsx (hunk 2 above)
Comment (pollen8) [reply src/ScatterPlot.tsx]: I've removed that line now as it wasn't used

**Pull request merged**